resources/styles.css

```diff
@@ -30,14 +30,21 @@
     background-color: #af0000;
 }
 
+.collapsible{
+ margin-top: 0px;
```

Comment:
Per css convention, a 0 value should not have type (`px`), so just write `margin-top: 0;`